pandas/tests/groupby/aggregate/test_aggregate.py

```diff
@@ -1063,6 +1063,193 @@ def test_groupby_get_by_index():
     pd.testing.assert_frame_equal(res, expected)
 
 
+def test_groupby_single_agg_numeric_col():
+    """
```

Comment:
pls add the issue number as a comment.

you don't really need the doc-string; the test name should be descriptive enough